src/picomsg/src/dummy_oper_db_client/dummy_oper_db_client.cpp

```diff
@@ -170,6 +264,13 @@ oper_db_client_db_obj_t* DummyOperDBClient::do_db_obj_create(enum LIBOPER_DB_CLI
 bool DummyOperDBClient::do_set(oper_db_client_db_obj_t* obj, int flags) {
 	c_object_wrapper<oper_db_client_db_obj_t> _dtor {obj};
 
+	reinterpret_cast<DummyOperDBObject*>(obj)->log(flags);
+	return true;
+
+}
+bool DummyOperDBClient::do_del(oper_db_client_db_obj_t* obj, int flags) {
+	c_object_wrapper<oper_db_client_db_obj_t> _dtor {obj};
```

Comment:
`DummyOperDBObject::log()` will write "client_set" at the beginning. Given this is a delete operation it should audit "client_del" instead.